paddle/pybind/pybind.cc

```diff
@@ -25,9 +26,143 @@ namespace pd = paddle::framework;
 
 USE_OP(add_two);
 
+struct PlaceDebugString : public boost::static_visitor<std::string> {
```

Comment:
- It seems that `PlaceDebugString` should be in `platform/place.h`, but not here.

- The usage of `PlaceDebugString` in this PR is not for *debugging*, so the name shouldn't contain `Debug`.   `PrintPlace` seems more accurate.

- `PrintPlace` can call `std::ostream &operator<<(std::ostream &, const Place &);` in `platform.place.h`, or vice versa, to ensure the consistent representation of Places.